config/settings/development.yml

```diff
@@ -6,3 +6,4 @@ symphony_catalogdump: '/s/sirsi/Unicorn/Bin/catalogdump -om -kc -h -z -j -n dump
 # Note: the dumpjunktag.Bibframe file is located on the Symphony server under /s/sirsi/Unicorn/Custom/.
 #       It contains the tag listing '001' which instructs the catalogdump to remove all '001' fields of
 #       the MARC record. The -kc flag then adds back in an '001' field with the ckey as the value
+base_uri: 'http://ld4p-dev.stanford.edu/'
```

Comment:
Is there also a `config/settings/test.yml` file that would be used in rspec (on travis)?